src/java.base/share/classes/java/lang/FdLibm.java

```diff
@@ -451,7 +453,9 @@ static double compute(double x) {
          */
         private static final double
             pio4  =  0x1.921fb54442d18p-1,  // 7.85398163397448278999e-01
-            pio4lo=  0x1.1a62633145c07p-55, // 3.06161699786838301793e-17
+            pio4lo=  0x1.1a62633145c07p-55; // 3.06161699786838301793e-17
+        @Stable
+        private static final double
             T[] = {
```

Comment:
The legacy **C**‑like array field syntax is confusing in **Java**, and it’s not used for the `@Stable int[]` arrays below.

```suggestion
        @Stable
        private static final double[] T = {
```

--------------------------------------------------------------------------------

Or at least put the `T[]` on the same line so that it’s clear why this field has `@Stable`, since regular `static final double`s are already constant folded and @wenshao’s comment[^1] confused me on the “**Conversation**” tab, since it was missing the `T[]` there.

```suggestion
        @Stable
        private static final double T[] = {
```

[^1]: https://github.com/openjdk/jdk/pull/26341#discussion_r2213012163